packages/block-library/src/post-terms/style.scss

```diff
@@ -1,3 +1,5 @@
 .wp-block-post-terms__separator {
 	white-space: pre-wrap;
+	// This block has customizable padding, border-box makes that more predictable.
+	box-sizing: border-box;
```

Comment:
This rule is currently only targeting the separator, so it looks like we need to add a separate rule to target the wrapper classname 🙂

<img width="1434" alt="image" src="https://user-images.githubusercontent.com/14988353/187056272-a4b3fcdc-d40c-453c-8f10-e89915752df4.png">

E.g.

```scss
.wp-block-post-terms {
	// This block has customizable padding, border-box makes that more predictable.
	box-sizing: border-box;
}
```

Reply:
Ahh, my sloppy mistake. I will correct that.